.github/workflows/cypress-tests.yaml

```diff
@@ -25,14 +32,35 @@ jobs:
       - name: Install and show git version
         run: apt-get -y install git && git --version
```

Comment:
why do you need git installation? Isn't it already present on all github runners?

Reply:
Good call, I didn't question the status quo, it's old and probably not needed at all. Let me see what happens if removed.

Reply:
it wasn't needed :pray: